src/generator.rs

```diff
@@ -6,16 +6,14 @@ use serde::{Deserialize, Serialize};
 
 use crate::{manifest::Manifest, package::PackageStore};
 
-/// The directory used for the generated code
-pub const BUILD_DIRECTORY: &str = "proto/build";
-
 /// The language used for code generation
 #[derive(
     Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize, clap::ValueEnum,
 )]
 #[serde(rename_all = "kebab-case")]
 pub enum Language {
     Rust,
```

Comment:
lets remove `Rust` from the language array and do:

```
pub enum Generator { Tonic, Protoc(Language) }
```

Reply:
this way it is impossible to render illegal states / we have typesafety, since:

- Rust code is always generated in cargos `OUT_DIR` (it is a non issue to call `buffrs gen --lang rust` / we dont need that feature!)